### PR TITLE
[feat] Added Datahub Utility + File Bundle 🚀

### PR DESCRIPTION
**Pull request by z3z1ma**
Adds datahub CLI as a utility allowing users to ingest metadata during meltano executions. Includes file bundle which creates the utility directory, includes a readme, and adds a dbt ingestion recipe for reference.

ie:

`tap-github target-bigquery dbt-bigquery:github datahub:dbt-ingest tap-dbt-artifacts target-bigquery`

`meltano invoke datahub ingest run --config path/to/recipe.yml`

### CODE REVIEW

File: _data/meltano/utilities/datahub/acryl.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  files:
+  - name: files-datahub
+    variant: z3z1ma
+settings:
+- name: gms_host
+  label: 
+    URL of DataHub GMS endpoint. Typically this is something like `your-server:8000`
+    where `your-server` is the server IP or domain name. If using metadata service authentication enabled in the image env (recommended), 
+    then this should instead point to your frontend which acts as a proxy for the GMS service. `datahub.yourdomain.com/api/gms`
+  kind: string
```
[the remark this answers is not in the document]
Comment (z3z1ma) [reply _data/meltano/utilities/datahub/acryl.yml:21]: Acknowledged! Thanks

**Comment by z3z1ma** [comment]
I looked at the jekyll template and saw how variant is used synonymously with github username in resolving the urls for that badges. I updated variant to reflect that. (I was originally thinking of it more like a maintainer alias and not important in its specificity)

This should be good to merge @pnadolny13 
I would be happy to scrap my bit of the from my meltano yml and source it from the hub once it is merged.

**Comment by z3z1ma** [comment]
Please squash when merging 🤣

**Review by pnadolny13 — CHANGES_REQUESTED** [review]

**Comment by pnadolny13** [comment]
>I looked at the jekyll template and saw how variant is used synonymously with github username in resolving the urls for that badges. I updated variant to reflect that. (I was originally thinking of it more like a maintainer alias and not important in its specificity)

@z3z1ma yeah thats mostly been our convention so far but it really shouldnt need to be tied to a github namespace. It would probably be better to use the repo url to build the badge in the future.

**Comment by pnadolny13** [comment]
@z3z1ma thanks for getting that updated! I have https://github.com/meltano/hub/issues/597#event-6821157228 to fix the CI validation errors youre seeing now so I'm comfortable merging this in as is.

**Pull request merged**